docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,377 @@
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access SQL Server Cloud](../../../img/database-access/guides/sqlserver_cloud.png)
+</ScopedBlock>
+
+This guide will focus on SQL Servers using self-hosted Active Directory
+authentication.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A SQL Server database with Active Directory authentication enabled.
+- A SQL Server network listener configured with a Certificate using Subject Alternative Names
+- A Windows machine joined to the same Active Directory domain as the database.
+- A Linux node with network access to the Active Directory.
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/7. Create a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+## Step 2/7. Configure a GPO to allow Teleport connections
+
+We need to configure a GPO to allow Teleport database sessions. This includes
+telling your computers to trust Teleport's CA and allowing certificate-based
+smart card authentication.
+
+### Export Teleport CA and CRL.
+
+The following step requires an existing cluster. If you don't already have a
+Teleport cluster up and running, see our general [Getting
+Started](../../try-out-teleport/introduction.mdx) guide.
+
+<Admonition type="note" title="Database CA Rotation">
+  These steps will need to be repeated if Teleport's database certificate
+  authority is rotated.
+</Admonition>
+
+1. Get the Teleport database CA certificate by running:
+
+```code
+$ tctl auth export --type=db-der > db-ca.cer
+```
+
+2. Get the Teleport database CRL by running:
+
+```code
+$ tctl auth crl --type=db > db-ca.crl
+```
+
+3. Transfer the `db-ca.cer` and `db-ca.crl` files to a Windows machine where you can manage your group policy.
+
+### Create a GPO and import the Teleport CA.
+
+<Admonition type="note" title="Domain Wide Policy">
+  For the purposes of this guide, we apply the GPO we are about to create to our
+  entire AD domain. In the case where you wish for only a subset of computers
+  within your AD domain to be accessible via Teleport, you should apply the GPO
+  to an OU that includes only such computers.
+</Admonition>
+
+1. Create a GPO with name like `Teleport DB Access`.
+
+```powershell
+$GPOName="Teleport DB Access"
+New-GPO -Name $GPOName | New-GPLink -Target $((Get-ADDomain).DistinguishedName)
+```
+
+2. Again open the `Group Policy Management` program, and on the left pane,
+   navigate to `$FOREST > Domains > $DOMAIN > Group Policy Objects`.
+3. Right click on the GPO you just made (`Teleport DB Access`), and select `Edit...`.
+4. In the group policy editor, select:
+
+```text
+Computer Configuration > Policies > Windows Settings > Security Settings > Public Key Policies
+```
+
+5. Right click on `Trusted Root Certification Authorities` and select `Import`.
+6. Click through the wizard, selecting your CA file.
```

Comment:
Not sure if I understood it correctly, but having a specific file location here implies how users transferred their CA files to the Windows machine (from the previous step). We could add a note on this, l[ike on the desktop access guide](https://github.com/gravitational/teleport/blob/master/docs/pages/desktop-access/active-directory-manual.mdx?plain=1#L201). What do you think?

Reply:
Just wonder if we can specify the filename because we have `db-ca.cer` and `db-ca.crl` so at first glance it might be consign because both files have the `-ca` suffix.